src/generator/css/base.less

```diff
@@ -645,17 +645,37 @@ body .content .card-list {
 }
 
 .post-list {
-    @apply grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-8;
+    @apply grid ;
 
     .post {
-        @apply mb-16 align-text-bottom flex flex-col;
+        @apply mb-16  align-text-bottom grid grid-cols-1 md:grid-cols-10 gap-10 ;
```

Comment:
Code style note please strip whitespaces between last declaraction and the ; character